### ISSUE TITLE
Integration of addon-storyshots with angular-cli project

### ISSUE THREAD

**Issue by x1c0** [storybooks/storybook#3290]
I'm trying to integrate addon-storyshots in an angular-cli project but I can't make it work.
I setup a new angular-cli project, I integrated with storybook and added the storyshots addon to serve as playground.

### Steps to reproduce

1. clone/download my repo -> https://github.com/x1c0/angular-storybook-jest
2. Run `npm run storybook` on one terminal.
3. Run `npm run jest` on another terminal.

The snapshots are not being captured, should they be? Am I missing any configuration? I followed all the addon documentation steps.

Running the `npm run jest` just hangs here, no error:
![image](https://user-images.githubusercontent.com/2421483/37912424-ccde73f4-3112-11e8-8684-313caf30d6a1.png)

### Please specify which version of Storybook and optionally any affected addons that you're running

- storybook/angular: "^3.4.0-rc.3",
- storybook/addon-notes: "^3.4.0-rc.3",
- storybook/addon-actions: "^3.4.0-rc.3",
- storybook/addon-links: "^3.4.0-rc.3",
- storybook/addons: "^3.4.0-rc.3",
- storybook/addon-storyshots: "^3.4.0-rc.3",

### Affected platforms 

Mac OS

**Comment by igor-dv** [comment]
Will check it today

**Comment by igor-dv** [comment]
I've managed to make it work with multiSnapshots:

```js
import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';

initStoryshots({
    test: multiSnapshotWithOptions({}),
});
```

Also, there is a problem with the inner `Integrity Test`, that takes too much time (#3301). I think I will remove them as a temporal solution.

**Comment by igor-dv** [comment]
@Hypnosphi can't we release it as a patch for 3.4 as well ?

**Comment by Hypnosphi** [comment]
Looks we've actually released it as 3.4 but it didn't get into changelog

cc @shilman

**Comment by Hypnosphi** [comment]
Now released as [`3.4.4`](https://github.com/storybooks/storybook/releases/tag/v3.4.4)
